Snowdog_Components/components/Organisms/quantity-update/quantity-update.config.js

```diff
@@ -38,6 +38,12 @@ module.exports = {
         qtyClass: 'input',
         qtyLabel: 'Qty'
       }
-    }
+    },
+    {
+      name: 'no-buttons-m',
```

Comment:
what do you think about changing to a more descriptive name that better explains what the purpose is? which is to not display buttons from $screen-m and above